app/assets/stylesheets/layout.scss

```diff
@@ -71,7 +79,7 @@ body{
       @include default-borders(1px);
 
       &.delete{
-        background: #cdacac;
+        background: #e4d2d2;
```

Comment:
Text color here is `#333`, so the original background color had a ratio of 6.1:1 (AA but not AAA). This is 8.7:1.